apps/language_models/src/pipelines/falcon_pipeline.py

```diff
@@ -189,12 +215,18 @@ def compile_falcon(self):
                 bytecode_stream = BytesIO()
                 module.operation.write_bytecode(bytecode_stream)
                 bytecode = bytecode_stream.getvalue()
-                del module
 
                 print(f"[DEBUG] writing mlir to file")
-                with open(f"{self.model_name}.mlir", "wb") as f_:
+                with open(f"{self.falcon_mlir_path}_elided.mlir", "w") as f_:
                     with redirect_stdout(f_):
-                        print(module.operation.get_asm())
+                        print(module.operation.get_asm(large_elements_limit=4))
```

Comment:
For debugging purpose it's fine but I don't think we should enable dumping elided IR via any control path.